public/gh-reroute/404.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>AICS 3D Cell Viewer</title>
+    <style>
+      html {
+        box-sizing: border-box;
+      }
+      html,
+      body,
+      main {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        padding: 0;
+        background-color: #222222;
+        color: #bfbfbf;
+        font-family: "Open Sans", sans-serif;
+      }
+
+      #error-page-404 {
+        margin-top: 20px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+
+        a {
+          color: #0099ff;
+
+          &:hover,
+          &:focus-within {
+            text-decoration: underline;
+          }
+        }
+      }
+    </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
```

Comment:
it doesn't seem like you're using all the weights here, could removing these cut down on the load time?

Reply:
That's a good idea, thank you!